sdk/cosmos/azure-cosmos-spark_3_2-12/pom.xml

```diff
@@ -709,6 +527,172 @@
       </build>
     </profile>
 
+    <profile>
```

Comment:
is this just moving profile to the correct section from above?

Reply:
Correct, this was just moving to profile to the correct section. For some reason this turned into a very misleading git diff